core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -293,6 +293,27 @@ private[data] trait EitherTFunctions {
     def apply[E, A](opt: Option[A], ifNone: => E)(implicit F: Applicative[F]): EitherT[F, E, A] =
       EitherT(F.pure(Either.fromOption(opt, ifNone)))
   }
+
+  /**  If the condition is satisfied, return the given `A` in `Right`
+    *  lifted into the specified `Applicative`, otherwise, return the
+    *  given `E` in `Left` lifted into the specified `Applicative`.
+    *
+    * {{{
+    * scala> val userInput = "hello world"
```

Comment:
I think we need to import `cats.data.EitherT` and (for the `Applicative[Future]`) `cats.implicits._`, `scala.concurrent.Future` and an `ExecutionContext`. Newlines also need a `|` at the start of the line (see for example the `toNested` example).

It seems that this `cond` example wasn't tested by travis (if I haven't missed it), maybe because the format wasn't right ?

The easiest way I found to quickly test a doctest example is to paste it in a fresh `console` (otherwise it is easy to forget some of the imports).

Apart of the example :+1: